doc/cephfs/health-messages.rst

```diff
@@ -240,3 +240,15 @@ other daemons, please see :ref:`health-checks`.
   Description
     All MDS ranks are unavailable resulting in the file system to be completely
     offline.
+
+``MDS_CLIENTS_LAGGY``
+----------------------------
+  Message
+    "Client *ID* is laggy; not evicted because some OSD(s) is/are laggy"
+
+  Description
+    OSD(s) can be laggy at times and this can lead to laggy clients(session might
+    get idle or cannot flush dirty data for cap revokes). This health warning
+    is generated when ``defer_client_eviction_on_laggy_osds`` is set to true
+    (default true), thus deferring clients eviction until OSD(s) return to
+    normal.
```

Comment:
This sounds like the client eviction is done after the OSDs are un-laggy, however, the client would be able to work as expected after the OSD is un-laggy.

Might be good to mention this with clarity.